README.md

```diff
@@ -0,0 +1,17 @@
+# Game8
+### Manifest
+Application contains files like **Images**,**index.html**,**Styles.css** & a **README.md** file., dosent require any installation and can run directly on any Browser Online.
+### About Project
+This Application is basically a Gaming site where an end user can access to **Blogs** , **Shopping** , Other Pages of the Application & Contact information of team.
+One can **play and Watch Live Game Streaming** , Can Participate in Upcoming Matches & watch the Current Teams participation., Provides a full history of Available
+Games,Live **Youtube Streams** and Current Winners.
+### Subscription
+Available **Free Subscription** to News Letter to get daily Notifications.
+### Link
+```
+git clone git@github.com:balaharikrishna/frontend-training-1.git
```

Comment:
change this to https link instead of ssh link

Reply:
changed to"https://github.com/balaharikrishna/frontend-training-1.git"